os/se/ameba/Kconfig

```diff
@@ -44,6 +44,13 @@ config HW_AES_ENC
 	---help---
 		Encrypts a data based on hardware.
 
+config HW_AES_GCM_ENC
+	bool "HW AES GCM encryption decryption"
```

Comment:
The config name is `ENC` but explanation says `encryption descryption`. What is correct?

Reply:
GCM support both enc and dec. But other config(AES, RSA) name has only ENC, so I used enc only in the same way.
If I need to change this, I'll change all config name in the other commit.